meta/types/bucket.go

```diff
@@ -75,8 +86,8 @@ func (b Bucket) GetUpdateSql() (string, []interface{}) {
 	cors, _ := json.Marshal(b.CORS)
 	lc, _ := json.Marshal(b.LC)
 	bucket_policy, _ := json.Marshal(b.Policy)
-	sql := "update buckets set bucketname=?,acl=?,policy=?,cors=?,lc=?,uid=?,versioning=? where bucketname=?"
-	args := []interface{}{b.Name, acl, bucket_policy, cors, lc, b.OwnerId, b.Versioning, b.Name}
+	sql := "update buckets set bucketname=?,acl=?,policy=?,cors=?,lc=?,uid=?,versioning=?,file_counts=?,last_usage_update_time=? where bucketname=?"
```

Comment:
You cannot update these meta in this sql. You'd better to update these columns when updating `usage`.

Reply:
Thanks @cxt90730 for reviewing my PR and your suggestions.
I will make a seperate branch for the bucket usage updating.
Just close this PR.